virttest/utils_libvirt/libvirt_monitor.py

```diff
@@ -53,7 +54,55 @@ def _check_items(value_type, tmp_domjobinfo):
     return tmp_domjobinfo
 
 
-def check_domjobinfo_output(params):
+def _get_value(expect_key, out):
+    """
+    Get value from out
+
+    :param expect_key: expect key
+    :param out: domain job info
+    :retrun: the value of expect key
+    """
+    for line in out.splitlines():
+        key = line.split(':')[0]
+        if key == expect_key:
+            value, unit = line.strip().split(':')[1].strip().split(' ')
+            if unit == "MiB":
+                value = round(float(value) / 1024, 3)
+            return float(value)
+    return None
+
+
+def _check_sum_items(line, key, tmp_domjobinfo, domjobinfo_out):
+    """
+    Check the items that need to be summed
+
+    :param line: one line of domjobinfo output
+    :param key: one key of domjobinfo output
+    :param tmp_domjobinfo: expect domain job info
+    :param domjobinfo_out: the output of virsh.domjobinfo command
+    :return: new domain job info
+    """
+    if "sum_items" in tmp_domjobinfo and key in tmp_domjobinfo["sum_items"]:
+        tmp_value = tmp_domjobinfo["sum_items"].get(key)
+        left, right = tmp_value.split('+')
+        left_value = _get_value(left, domjobinfo_out)
+        right_value = _get_value(right, domjobinfo_out)
+        value = _get_value(key, domjobinfo_out)
+        sum_value = round((left_value + right_value), 3)
+        if not math.isclose(sum_value, value, abs_tol=0.01):
+            raise exceptions.TestFail("'%s' is not matched expect '%s'" %
+                                      (sum_value, value))
+        else:
+            del tmp_domjobinfo["sum_items"][key]
+    return tmp_domjobinfo
+
+
+def check_domjobinfo_output(vm_name,
+                            expected_domjobinfo={},
+                            expected_domjobinfo_complete={},
```

Comment:
Plz update docstrings for these parameters, thx!

Reply:
Updated, thanks.